### ISSUE TITLE
panic: runtime error: index out of range [0]

### ISSUE THREAD

**Issue by slayer** [dty1er/kubecolor#15]
`kubecolor describe node xxx` 

```
Capacity:
  attachable-volumes-gce-pd:panic: runtime error: index out of range [0] with length 0

goroutine 6 [running]:
github.com/dty1er/kubecolor/printer.(*DescribePrinter).Print(0xc000100f08, 0x535320, 0xc00000e030)
        /home/vlad/dev/oss/kubecolor/printer/describe.go:93 +0xc84
github.com/dty1er/kubecolor/printer.Print(0x535320, 0xc00000e030, 0x535340, 0xc00000e018, 0xc000018980, 0x1)
        /home/vlad/dev/oss/kubecolor/printer/printer.go:81 +0x292
github.com/dty1er/kubecolor/command.Run.func7()
        /home/vlad/dev/oss/kubecolor/command/runner.go:79 +0x97
github.com/dty1er/kubecolor/command.runAsync.func1(0xc000076810, 0xc000016150)
        /home/vlad/dev/oss/kubecolor/command/runner.go:95 +0x27
created by github.com/dty1er/kubecolor/command.runAsync
        /home/vlad/dev/oss/kubecolor/command/runner.go:94 +0x6e
```

**Comment by dty1er** [comment]
Ah I found the root cause let me fix...

**Comment by dty1er** [comment]
I submitted a PR to fix the root cause on kubernetes side here: https://github.com/kubernetes/kubernetes/pull/95639

Now I will start working on implementing work-around on kubecolor side

**Comment by dty1er** [comment]
@slayer 
Hi, I confirmed this problem does not happen on the latest kubectl and it's fixed by kubectl 1.17.
I guess you're using kubectl 1.16 or lower right? If so, it's currently [not supported by kubernetes side](https://kubernetes.io/docs/setup/release/version-skew-policy/) so can you update your kubectl and try again?

**Comment by slayer** [comment]
Yes, I was using v1.16.3, updated now to v1.19.3. No issues now.
 Thanks
